src/main/java/com/minecolonies/coremod/commands/CitizenInfoCommand.java

```diff
@@ -0,0 +1,175 @@
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+
+public class CitizenInfoCommand extends AbstractSingleCommand
+{
+    public static final  String       DESC                            = "citizenInfo";
+    private static final String       CITIZEN_DESCRIPTION             = "§2ID: §f %d §2 Name: §f %s";
+    private static final String       NO_COLONY_CITIZEN_FOUND_MESSAGE = "No citizen %d found in colony %d.";
+    private static final String       CITIZEN_DATA_NULL               = "Couldn't find citizen client side representation of %d in %d";
+    private static final String       ENTITY_CITIZEN_NULL             = "Couldn't find entity of %d in %d";
+    private static final String       COLONY_NULL                     = "Couldn't find colony %d";
+    private static final String       CITIZEN_LEVEL_AND_AGE           = "§2Level: §f%s §2Age: §f%s §2Experience: §f%s";
+    private static final String       CITIZEN_SKILLS                  = "§2Charisma: §f%s §2Dexterity: §f%s §2Endurance: §f%s\n§2Intelligence: §f%s §2Strength: §f%s";
+    private static final String       CITIZEN_JOB                     = "§2Job: §f%s";
+    private static final String       CITIZEN_JOB_NULL                = "§2Job: §fUnemployed";
+    private static final String       CITIZEN_HEALTH                  = "§2Health: §f%s §2Max Health: §f%s";
+    private static final String       CITIZEN_DESIRED_ACTIVITY        = "§2Desired activity: §f%s §2Current Activity: §f%s";
+    private static final String       CITIZEN_HOME_POSITION           = "§2Home position: §4x=§f%s §4y=§f%s §4z=§f%s";
+    private static final String       CITIZEN_WORK_POSITION           = "§2Work position: §4x=§f%s §4y=§f%s §4z=§f%s";
+    private static final String       CITIZEN_POSITION                = "§2Citizen position: §4x=§f%s §4y=§f%s §4z=§f%s";
+    private static final String       CITIZEN_WORK_POSITION_NULL      = "§2Work position: §4No work position found!";
+    private static final String       CITIZEN_NO_ACTIVITY             = "§4No activity is being desired or executed!";
+
+
+
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public CitizenInfoCommand(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @NotNull
+    @Override
+    public String getCommandUsage(@NotNull final ICommandSender sender)
+    {
+        return super.getCommandUsage(sender) + "<ColonyId> <CitizenId>";
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "execute" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)